docs/DeveloperGuide.md

```diff
@@ -691,65 +696,64 @@ Use case ends.
 
 Use case ends.
 
-**Extensions**
+**Extensions:**
 
-1a. User is unable to access the main window. <br>
-1a1. System displays an error message. <br>
-2a. System recognizes that user has entered invalid task details. <br>
-2a1. System displays an error message indicating the task detail error(s). <br>
-
-Use case ends.
+* 1a. User is unable to access the main window. <br>
+  * 1a1. System displays an error message. <br>
+    Use case ends.
+* 2a. System recognizes that user has entered invalid task details. <br>
+  * 2a1. System displays an error message indicating the task detail error(s). <br>
+    Use case resumes from Step 2.
 
 **Use Case: Delete Internship Task From Main Window**
 
-**MSS**
+**MSS:**
 1. User accesses the main window. 
 2. User uses the command to specify which internship task to delete. 
```

Comment:
Specifying internship entry and task to delete from it
This appears for a few other MSS as well